server/game/cards/04-MM/SacroSaurus.js

```diff
@@ -16,6 +16,16 @@ class SacroSaurus extends Card {
                 }
             }
         });
+        this.play({
```

Comment:
You have to remove the fight effect.
![image](https://user-images.githubusercontent.com/2104075/80863802-f7c13f80-8c54-11ea-92e1-ccf5a6b856de.png)

Reply:
removed